### PR TITLE
[13.0][l10n_us_gaap] should use anglo saxon setting

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
cc @AdriaGForgeFlow @LoisRForgeFlow

### CODE REVIEW

**Review by AdriaGForgeFlow — APPROVED** [review]
LGTM :+1:

File: l10n_us_gaap/data/account_chart_template_data.xml [hunk 1]
```diff
@@ -7,6 +7,7 @@
         <field name="bank_account_code_prefix">1111</field>
         <field name="cash_account_code_prefix">1112</field>
         <field name="transfer_account_code_prefix">1119</field>
+        <field name="use_anglo_saxon" eval="True"/>
```
Comment (bodedra) [inline l10n_us_gaap/data/account_chart_template_data.xml]:
```suggestion
        <field name="use_anglo_saxon" eval="True" />
```

**Review by bodedra — APPROVED** [review]
Pre-commit minor change.

LGTM :+1:

**Review by max3903 — APPROVED** [review]

**Pull request merged**